app/views/recipes/show.html.erb

```diff
@@ -0,0 +1,68 @@
+
+<div class="container">
+    <div class="card text-center ">
+        <h4 class="card-header" style="background: var(--primary-color); color: var(--white)">
+          <%= image_tag @recipe.image, class: "float-start bio-photo" if @recipe.image.present? %>
+          <%= @recipe.name %>
+        </h4>
+        <div class="row mx-5 mt-2">
+          <div class="col-md-9">
+              <p class="text-start"><i class="fa fa-clock-o" aria-hidden="true"></i> Preparation time: <%= @recipe.preparation_time %></p>
+              <p class="text-start"><i class="fa fa-clock-o" aria-hidden="true"></i> Cooking time: <%= @recipe.cooking_time %></p> 
+               <p class="text-start"><i class="fa fa-clock-o" aria-hidden="true"></i><u> Description</u>: <br> <%= @recipe.description %></p>
+          </div>
+          <% if current_user && @recipe.user_id == current_user.id %>
+            <div class="col-md-3">
+                <%= form_with model: @recipe, url: recipe_path(@recipe.id), method: :patch, local: true do |form| %>
+                  <div class="form-switch">
+                      <%= form.check_box :public, class: "form-check-input", onchange: 'this.form.submit()' %>
```

Comment:
Hey guys, this check box is not performing as it should. When I click on the checkbox, it throws an error. [Check the video.](https://www.loom.com/share/4f8a9e01096f4610842700616d79bab3) Since the `update` is not the part of the crud you are supposed to perform on the recipe, you can either remove the intractability from the check box or if you want to have this as a feature, well in that case, you can just fix the error. 😊